CONTRIBUTING.md

```diff
@@ -41,9 +41,12 @@ $ npm run start
 
 ## Pull Requests
 
-We actively welcome your pull requests.
+We actively welcome your pull requests. 
```

Comment:
I know this isn't related, but would you mind changing this to an exclamation point while you're in there, and maybe even add to this sentence a little? Reminds of the "IT IS YOUR BIRTHDAY." sign from The Office.